Automattic/wp-calypso
# Add locale switcher to Jetpack pricing page

### Changes proposed in this Pull Request

This PR adds the ability to switch locale in the Jetpack pricing page.

It mimic the implementation of [Jetpack.com](http://jetpack.com/).

### Testing instructions

- Download the PR and run cloud
- Visit `/pricing`
- Check that you can see the language selector in the footer
- Verify that you can switch language

### Screenshots
<img width="1032" alt="Screen Shot 2022-02-03 at 3 35 20 PM" src="https://user-images.githubusercontent.com/1620183/152424599-b4f68ede-db44-4e54-99b9-359ec47452b4.png">
